src/ui/popup.js

```diff
@@ -213,6 +213,15 @@ class Popup extends Evented {
     }
 
     /**
+     * Returns the `Popup`'s HTML element.
+     * @returns {HTMLElement} element
+     */
+    getElement() {
+        return this._content ? this._content : null;
+
+    }
```

Comment:
🤔  there's a a slight mismatch here between the element returned by `getElement()` -- which is the wrapping `<div class="mapboxgl-popup-content">...</div>` -- versus the element provided by, say, `setDOMContent()` / `setHTML()` / `setText()`, which would set the _content_ of that div.

If someone does `setHTML(<div class="my-popup">...</div>)`, they might reasonably expect to get the `"my-popup"` div back from `getElement()`.  But since the the content provided by the setters need not be a single element, we can't use that as the return value for `getElement()`.  Maybe the mismatch is fine, but we specify it in the docs?

Reply:
ah you're right @anandthakker – it is weird especially bc this will include the little close button we include by default 💭

Reply:
removing `Popup` portion of this PR – will revisit if we get user requests.